resolvers.js

```diff
@@ -17,6 +19,40 @@ export const resolvable = collection =>
     },
   });
 
+export const paginatable = collection =>
```

Comment:
I would guess that pagenable would be better but not sure.

Reply:
Oh agree, Webstorm didn't recognize `paginable` as a valid word so I thought it was wrong, but searching online it seems just as valid as `paginatable`.